src/Illuminate/Auth/Passwords/PasswordBroker.php

```diff
@@ -35,15 +45,15 @@ class PasswordBroker implements PasswordBrokerContract
     /**
      * Create a new password broker instance.
      *
-     * @param  \Illuminate\Auth\Passwords\TokenRepositoryInterface  $tokens
+     * @param  \Illuminate\Contracts\Foundation\Application  $app
```

Comment:
We shouldn't inject the entire application here just to get the key. Just inject the key.